example/facebook_share.js

```diff
@@ -94,10 +94,8 @@ exports.window = function (value) {
 
 	requestDialog.addEventListener('click', function () {
 		fb.presentSendRequestDialog({
-			message: 'Go to https://appcelerator.com/'
-		}, {
-			data: '{\'badge_of_awesomeness\':\'1\',' +
-				'\'social_karma\':\'5\'}'
+			message: 'Go to https://appcelerator.com/',
+            data: '{\'badge_of_awesomeness\':\'1\',' + '\'social_karma\':\'5\'}'
```

Comment:
The `data` is stringified JSON here, which is not correct. The old examples were wrong there too. Please see my example again and the example in the [readme](https://github.com/appcelerator-modules/ti.facebook#send-requests-dialog) how it should look like

```js
fb.presentSendRequestDialog({
  message: 'Go to https://appcelerator.com/',
  data: { badge_of_awesomeness: '1', social_karma: '5' }
});
```

There are also other examples [here](https://github.com/appcelerator-modules/ti.facebook/blob/46d53182e5a44ae9a6eca1963f05847638a5c902/android/example/facebook_share_dialog.js#L77) and [here](https://github.com/appcelerator-modules/ti.facebook/blob/3f10c4ab59ddcfb26a22b0e68a1ffbc07aac418a/android/example/facebook_publish_stream.js#L153) that need to be adjusted as well.